Package.swift

```diff
@@ -1,12 +1,15 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.1
```

Comment:
Is it necessary to increase swift-tools-version version?

Reply:
Not necessary - just trying to keep things up-to-date. Any issue with 5.1? (Same applies to bumping Xcode and iOS versions)

Reply:
I suppose it will break compatibility with 4.0 which is nice to have. From docs:

> The Swift tools version declares the version of the PackageDescription library, the minimum version of the Swift tools and Swift language compatibility version to process the manifest, and the minimum version of the Swift tools that are needed to use the Swift package.

So looks like changing this will prevent code to work on Swift 4.0

Reply:
Oh, I think this is swift tools version (Xcode) and not swift. So if you have Xcode 11 then you should be alright. 

That said, I don’t have strong opinion here. Lemme revert this.